tableschema_spss/storage.py

```diff
@@ -42,60 +33,21 @@ def __init__(self, base_path=None):
             self.__reindex_buckets()
 
     def __repr__(self):
+        """https://github.com/frictionlessdata/tableschema-spss-py#storage
+        """
         return 'Storage <{}>'.format(self.__base_path)
 
-    def __reindex_buckets(self):
-        self.__buckets = self.__list_bucket_filenames()
-
-    def __list_bucket_filenames(self):
-        '''Find .sav files at base_path and return bucket filenames'''
-        return [f for f in os.listdir(self.__base_path) if f.endswith(('.sav', '.zsav'))]
-
-    def __get_safe_file_path(self, bucket, check_exists=False):
-        '''Return a file_path to `bucket` that doesn't traverse outside the base
-        directory.'''
-
-        if self.__base_path:
-            # base_path exists, so `bucket` is relative to base_path
-            filename = mappers.bucket_to_filename(bucket)
-            file_path = os.path.join(self.__base_path, filename)
-            norm_file_path = os.path.normpath(file_path)
-            if not norm_file_path.startswith(os.path.normpath(self.__base_path)):
-                raise RuntimeError('Bucket name "{}" is not valid.'.format(bucket))
-        else:
-            norm_file_path = bucket
-
-        if check_exists and not os.path.isfile(norm_file_path):
-            # bucket isn't a valid file path, bail
-            raise RuntimeError('File "{}" does not exist.'.format(norm_file_path))
-
-        return norm_file_path
-
     @property
     def buckets(self):
-        '''List all .sav and .zsav files at __base_path.
```

Comment:
What's the thinking behind removing the docstring comments?

Reply:
There was a long way to get to it but I've decided to have one point of truth for all documentation across implementation stack. For now it's already around 20 repositories with very limited resources to maintain it. And docs was starting to be badly unsync in different places. Because users also needs to have API references now I put all documentation to readmes (not the code - having only link in the code) like this:
- https://github.com/frictionlessdata/datapackage-py#packagedescriptor-base_pathnone-strictfalse
- https://github.com/frictionlessdata/tableschema-js#async-tableloadsource-schema-strictfalse-headers1

For storages there is no need to describe all methods because it's described in https://github.com/frictionlessdata/tableschema-py#interfacestorageoptions but it seems I've missed that `storage.buckets` has slightly different symantics so fixed:
- https://github.com/frictionlessdata/tableschema-spss-py/tree/specs-v1#storagebase_pathnone